pubsub/rocketmq/rocketmq_test.go

```diff
@@ -26,62 +27,193 @@ import (
 
 func getTestMetadata() map[string]string {
 	return map[string]string{
-		"nameServer":         "127.0.0.1:9876",
+		"nameServer":         "172.16.101.223:9876,172.16.101.224:9876",
```

Comment:
Please don't hardcode IPs that may not exist. Use an environmental variable to pass the IP, with fallback to `127.0.0.1:9876`?

Reply:
ok